test/index.js

```diff
@@ -479,8 +860,37 @@ describe('def', function() {
 
     throws(function() { even(0.5); },
            errorEq(TypeError,
-                   '‘even’ expected a value of type Integer ' +
-                   'as its first argument; received 0.5'));
+                   'Invalid value\n' +
+                   '\n' +
+                   'even :: Integer -> Boolean\n' +
+                   '        ^^^^^^^\n' +
+                   '           1\n' +
+                   '\n' +
+                   '1)  0.5 :: Number\n' +
+                   '\n' +
+                   'The value at position 1 is not a member of Integer.\n'));
+
+    //  fromMaybe :: a -> anonMaybe a
+    var fromMaybe = def('fromMaybe', {}, [a, anonMaybe(a), a],
+                                         function(x, maybe) {
+      return maybe.isJust ? maybe.value : x; }
+    );
```

Comment:
Let's go with this formatting:

```javascript
//  fromMaybe :: a -> anonMaybe a
var fromMaybe =
def('fromMaybe',
    {},
    [a, anonMaybe(a), a],
    function(x, maybe) { return maybe.isJust ? maybe.value : x; });
```

When line wrapping of arguments is required, I like each argument to begin at the same column position. It makes it easier to see how things are nested.